src/lollipopChart.js

```diff
@@ -362,7 +388,7 @@ var LollipopChart = function (selection) {
   chart.lollipopRadius = function(_) {
     if(!arguments.length) return lollipopRadius;
     lollipopRadius = _;
-    chartGutter = _;
+    //chartGutter = _; //is this a bug?
```

Comment:
This was to make the top gutter accomodate for the lollipop circle. I think it's still needed because it looks like now the gutter is cutting off the top of the lollipops.